README.md

```diff
@@ -22,12 +22,21 @@ from [kcdc.info](http://www.kcdc.info/), and returns results about these talks.
 
 #### Running
 
+**Prerequisites:**
+* [Maven 3.x](https://maven.apache.org/index.html)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+
 Assemble the service:
 
 ```bash
 mvn clean package
 ```
 
+Optionally, you can add `-DskipTests` if you notice test failures when running `mvn clean package`:
```

Comment:
yeah, that seems like a good approach! I'll see if I can get those random thread failures fixed! (I think it comes from the wiremock rule) but I haven't looked 2 deep